robot_radius: 0.2  #用于全局规划的机器人半径，必须设置为车体的短边长度
map_inflation_radius: 1.0  #全局costmap的膨胀半径，必须大于机器人半径，否则路径无法规划
robot_radius_a : 0.2  #用于局部规划的机器人半径，这个应该根据实际情况调整，最保险的设置为车体对角线半径
